### PR TITLE
Architecture changes

### PR DESCRIPTION
**Pull request by yogeshbdeshpande**
Use correct terminology(Transparency Registry) reflecting the Transparency Architecture as against an implementation terminology of ledger.

### CODE REVIEW

**Comment by yogeshbdeshpande** [comment]
@henkbirkholz Can you please review and let me know, your thoughts on the PR?

File: draft-birkholz-scitt-architecture.md [hunk 1]
```diff
@@ -170,24 +178,24 @@ Envelope:
 
 Feed:
 
-: An identifier chosen by the Issuer for the Artifact. For every Issuer and Feed, the Ledger on a Transparency Service contains a sequence of Claims about the same Artifact.
+: An identifier chosen by the Issuer for the Artifact. For every Issuer and Feed, the Registryon a Transparency Service contains a sequence of Claims about the same Artifact.
```
Comment (joshuagl) [inline draft-birkholz-scitt-architecture.md]: missing space
```suggestion
: An identifier chosen by the Issuer for the Artifact. For every Issuer and Feed, the Registry on a Transparency Service contains a sequence of Claims about the same Artifact.
```

File: draft-birkholz-scitt-architecture.md [hunk 2]
```diff
@@ -170,24 +178,24 @@ Envelope:
...
 
 Receipt:
 
-: a Receipt is a special form of COSE countersignature for Claims that embeds cryptographic evidence that the Claim is recorded in the Ledger. It consists of a Ledger-specific inclusion proof, a signature by the Transparency Service of the state of the Ledger, and additional metadata (contained in the countersignature protected headers) to assist in auditing.
+: a Receipt is a special form of COSE countersignature for Claims that embeds cryptographic evidence that the Claim is recorded in the Registry . It consists of a Registry -specific inclusion proof, a signature by the Transparency Service of the state of the Registry , and additional metadata (contained in the countersignature protected headers) to assist in auditing.
 
 Registration:
 
-: the process of submitting a Claim to a Transparency Service, applying its registration policy, storing it in the Ledger and producing the Receipt returned to the submitter.
+: the process of submitting a Claim to a Transparency Service, applying its registration policy, storing it in the Registryand producing the Receipt returned to the submitter.
```
Comment (joshuagl) [inline draft-birkholz-scitt-architecture.md]: missing space here, too
```suggestion
: the process of submitting a Claim to a Transparency Service, applying its registration policy, storing it in the Registry and producing the Receipt returned to the submitter.
```

File: draft-birkholz-scitt-architecture.md [hunk 3]
```diff
@@ -227,9 +235,10 @@ Reputable Issuers are thus incentivized to carefully review their Statements bef
                       v        signature  |     |
                     Claim  <--------------/     |
                       |                         |
-                      |   Receipt   +--------+  |
-Transparency ->       +-------------| Ledger |  /
-Service               |             +--------+ X
+                      |  Claim      +---------+ |
+                      |------------>| Trans.  | |
+Transparency ->       +<------------| Regisrty| /
```
Comment (joshuagl) [inline draft-birkholz-scitt-architecture.md]: typo: Regisrty -> Registry
```suggestion
Transparency ->       +<------------| Registry| /
```

File: draft-birkholz-scitt-architecture.md [hunk 4]
```diff
@@ -487,14 +496,14 @@ Multiple SCITT instances, governed and operated by different organizations.
 
 For example,
 - a small, simple SCITT instance may keep track specifically of the software used for operating SCITT services.
-- an air-gapped data center may operate its own SCITT Ledger to retain full control and auditing of its software supplies.
+- an air-gapped data center may operate its own SCITT Registry to retain full control and auditing of its software supplies.
 
 How?
-- Policy-based. Within an organization, local Verifiers contact an authoritative SCITT that records the latest policies associated with classes of Artifacts; these policies indicate which Issuers and Ledgers are trusted for verifying signed Transparent Claims for these Artifacts.
+- Policy-based. Within an organization, local Verifiers contact an authoritative SCITT that records the latest policies associated with classes of Artifacts; these policies indicate which Issuers and Registeries are trusted for verifying signed Transparent Claims for these Artifacts.
```
Comment (joshuagl) [inline draft-birkholz-scitt-architecture.md]: typo: Registeries -> Registries
```suggestion
- Policy-based. Within an organization, local Verifiers contact an authoritative SCITT that records the latest policies associated with classes of Artifacts; these policies indicate which Issuers and Registries are trusted for verifying signed Transparent Claims for these Artifacts.
```
Registries

File: draft-birkholz-scitt-architecture.md [hunk 5]
```diff
@@ -487,14 +496,14 @@ Multiple SCITT instances, governed and operated by different organizations.
...
 
 - Other federation mechanisms?
 
-We'd like to attach multiple Receipts to the same signed Claims, each Receipt endorsing the Issuer signature and a subset of prior Receipts. This involves down-stream Ledgers verifying and recording these Receipts before issuing their own Receipts.
+We'd like to attach multiple Receipts to the same signed Claims, each Receipt endorsing the Issuer signature and a subset of prior Receipts. This involves down-stream Registeries verifying and recording these Receipts before issuing their own Receipts.
```
Comment (joshuagl) [inline draft-birkholz-scitt-architecture.md]: typo: Registeries -> Registries
```suggestion
We'd like to attach multiple Receipts to the same signed Claims, each Receipt endorsing the Issuer signature and a subset of prior Receipts. This involves down-stream Registries verifying and recording these Receipts before issuing their own Receipts.
```

**Review by joshuagl — COMMENTED** [review]
Ledger does seem prescriptive, and could be read as implying specific implementation technologies. Registries is more generic (though Cloud folks might disagree).

**Pull request merged**